<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSetting</class>
 <widget class="QDialog" name="FormSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="settingOptimisationTab">
      <attribute name="title">
       <string>Оптимизация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Метод:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="methodOptimisatonComboBox">
           <item>
            <property name="text">
             <string>Хука-Дживса</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Квазиньютоновский</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Наискорейшего спуска</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Функция ошибки:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="errorAbsoluteRadioButton">
           <property name="text">
            <string>Абсолютная</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">errorButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="errorRelativeRadioButton">
           <property name="text">
            <string>Относительная</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">errorButtonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Погрешность ошибки:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="errorEpsSpinBox">
           <property name="suffix">
            <string/>
           </property>
           <property name="prefix">
            <string>1e-</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>30</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Количество итераций:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="iterationCountSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="reportSaveCheckBox">
         <property name="text">
          <string>Сохранять отчёт</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Формат отчёта:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="reportFormatHtmlRadioButton">
           <property name="text">
            <string>HTML</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">reportFormatButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="reportFormatTextRadioButton">
           <property name="text">
            <string>Текст</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">reportFormatButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="reportFormatXmlRadionButton">
           <property name="text">
            <string>XML</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">reportFormatButtonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="errorButtonGroup"/>
  <buttongroup name="reportFormatButtonGroup"/>
 </buttongroups>
</ui>
